ocp_resources/deployment.py

```diff
@@ -32,13 +32,13 @@ def scale_replicas(self, replica_count=int):
         LOGGER.info(f"Set deployment replicas: {replica_count}")
         return self.update(resource_dict=body)
 
-    def wait_for_replicas(self, deployed=True, timeout=TIMEOUT):
+    def wait_for_replicas(self, timeout=TIMEOUT, deployed=True):
         """
         Wait until all replicas are updated.
 
         Args:
-            deployed (bool): True for replicas deployed, False for no replicas.
             timeout (int): Time to wait for the deployment.
+            deployed (bool): (Deprecated) True for replicas deployed, False for no replicas.  # TODO: Deprecated
```

Comment:
Add Deprecated warrning?

Reply:
I do not think we should remove this; a user should request an expected status.